helios-system-tests/src/main/java/com/spotify/helios/system/MultiplePortJobTest.java

```diff
@@ -92,14 +92,22 @@ public void test() throws Exception {
 
       final JobId jobId2 = createJob(testJobName + 2, testJobVersion, BUSYBOX, IDLE_COMMAND,
                                      EMPTY_ENV, ports2);
-
       assertNotNull(jobId2);
       deployJob(jobId2, testHost());
       final TaskStatus firstTaskStatus2 = awaitJobState(client, testHost(), jobId2, RUNNING,
                                                         LONG_WAIT_SECONDS, SECONDS);
 
-      assertEquals(expectedMapping1, firstTaskStatus1.getPorts());
-      assertEquals(expectedMapping2, firstTaskStatus2.getPorts());
+      // Verify we allocated dynamic ports from within the specified range.
+      assertTrue(portRange.contains(firstTaskStatus1.getPorts().get("foo").getExternalPort()));
+      assertTrue(portRange.contains(firstTaskStatus2.getPorts().get("foo").getExternalPort()));
```

Comment:
or i guess if the thing you are really testing is that `firstTaskStatus2.getPorts().get("foo").getExternalPort()` is in some expected range it would be nicer to reverse it like

```java
assertThat(blah.getExternalPort(), isIn(portRange))
```
which reads nicer when it fails (something like "expected N to be one of {1, 2, 3, 4}")